pmm-app/src/pmm-add-instance/panel.tsx

```diff
@@ -1,48 +1,53 @@
-import React, { useState } from 'react';
-import { Button } from 'antd';
+import React, { useEffect, useMemo, useState } from 'react';
 import { createBrowserHistory } from 'history';
 import { Router, Route } from 'react-router-dom';
+import { Button } from '@grafana/ui';
+import { cx } from 'emotion';
 import AddRemoteInstance from './components/AddRemoteInstance/AddRemoteInstance';
-import DiscoveryPanel from './components/DiscoveryPanel/DiscoveryPanel';
+import Discovery from './components/Discovery/Discovery';
 import { AddInstance } from './components/AddInstance/AddInstance';
-import './panel.scss';
-import '../shared/style.less';
-import '../shared/styles.scss';
+import { getStyles } from './panel.styles';
 
+const availableInstanceTypes = ['rds', 'postgresql', 'mysql', 'proxysql', 'mongodb', 'proxysql'];
 const history = createBrowserHistory();
+
 const AddInstancePanel = () => {
+  const styles = getStyles();
+
   const urlParams = new URLSearchParams(window.location.search);
   const instanceType = urlParams.get('instance_type') || '';
-  const availableInstanceTypes = ['rds', 'postgresql', 'mysql', 'proxysql', 'mongodb', 'proxysql'];
   const [selectedInstance, selectInstance] = useState({
     type: availableInstanceTypes.includes(instanceType) ? instanceType : '',
   });
 
-  const setSelectedInstance = (instance) => {
+  useEffect(() => {
     const url = new URL((window.location as unknown) as string);
```

Comment:
Do you really need to cast to unknown here if afterwards you are casting to string?

Reply:
Yep, it only works that way, to be honest, I just didn't want to spend time to find more elegant solution

Reply:
`const location = useLocation();` 
But it needs a router, unfortunately. 
LOL, I discovered we have the router, so consider using it instead.